demos/TV/TV/MainViewController.m

```diff
@@ -0,0 +1,22 @@
+#import "MainViewController.h"
+
+@implementation MainViewController {
+  UILabel *_label;
+}
+
+- (void)loadView {
+  [super loadView];
```

Comment:
loadView has some sharp edges.  
One is: if you override this method to create view manually you should not call super. You are responsible for creating the entire hierarchy and assigning it to the view controller's view property.